src/components/organisms/UiList/stories/ListBox.vue

```diff
@@ -0,0 +1,74 @@
+<template>
+  <UiList
+    v-bind="args"
+    class="list-box"
+  />
+  <div class="search-dropdown-input-row">
```

Comment:
in the design whole row is clickable/focused:
<img width="415" alt="image" src="https://github.com/infermedica/component-library/assets/37744371/0564e7a5-a91d-41e9-add6-9044fbb6b76d">

in storybook only checkbox:
<img width="174" alt="image" src="https://github.com/infermedica/component-library/assets/37744371/bba313e7-130b-4484-892b-0780fc6804a7">